src/op_analytics/cli/subcommands/pulls/defillama.py

```diff
@@ -22,50 +24,40 @@
 METADATA_TABLE = "defillama_stablecoins_metadata"
 
 
-def get_data(session, url):
-    start = time.time()
-    resp = session.request(
-        method="GET",
-        url=url,
-        headers={"Content-Type": "application/json"},
-    ).json()
-    log.info(f"Fetched from {url}: {time.time() - start:.2f} seconds")
-    return resp
+def process_breakdown_stables(
```

Comment:
now that I have been going over this data pull more carefully I see that the process_breakdown_stables function is doing two separate things, so it should be broken apart into two functions.  One is getting the metadata, the other one is getting the balances.   

At the time when we first coded this up we put it toghether somewhat quickly and were not very careful about things like this. Also the names we used could have been a lot better. For example, `balances` would be more indicative of what is going and therefore a better name than `breakdown`